### DISCUSSION TITLE
Larger Start Menu?

### DISCUSSION THREAD

**Discussion by AldrigGittan** [ramensoftware/windhawk-mods#1288] [Q&A]
Amazing tool! I'm searching for a mod/setting to enlarge the start menu. Perhaps double the size of what it is by default. What mod/setting should I use?

**Comment by m417z** [comment]
Check out the [Windows 11 Start Menu Styler](https://windhawk.net/mods/windows-11-start-menu-styler) mod. Take a look at [the Windows 11 start menu styling guide](https://github.com/ramensoftware/windows-11-start-menu-styling-guide) for target and style examples. If you need help with finding the right style, you can create an issue in the styling guide repository.
